subsurface-helper.cpp

```diff
@@ -77,7 +77,7 @@ void run_ui()
 	// To work around this we need to manually copy the components at install time
 	// to Contents/Frameworks/qml and make sure that we add the correct import path
 	const QStringList importPathList = engine.importPathList();
-	for (const QString &importPath: importPathList) {
+	for (QString importPath: importPathList) {
```

Comment:
The commit message doesn't explain why here you go the other way round: from reference to copy...?

Reply:
rebase error.

Reply:
needed because we modify importPath, this was just broken in an older commit and not fixed in the original commit.